vxyowsup/whatsapp.py

```diff
@@ -147,6 +151,21 @@ def onMessage(self, messageProtocolEntity):
 
     @ProtocolEntityCallback("receipt")
     def onReceipt(self, entity):
+        '''receives confirmation of delivery to human'''
+        # shady
+        # getting too many receipts
```

Comment:
Ah, I see from the tutorial that WhatsApp *does* require acks for events too, so this method is doing the correct thing. Maybe that is why the issue went away?